projects/workflows-creator/README.md

```diff
@@ -99,7 +99,7 @@ npm i @sourceloop/workflows-creator
 ### Usage
 
 The web component accepts all the same inputs and services as the regular Angular Module, but instead of passing them through bindings and DI, you pass them as properties of the element as shown below.
-NOTE: If `envIdentifier` is required it must be provided in the global `window` object as shown below, otherwise it will be rendered as undefined. This step is needed only in non-angular applications. For angular appplications it can be directly provided with the help of providers.
+NOTE: If `envIdentifier` is required it must be provided in the global `window` object as shown below, otherwise it will be rendered as undefined. This step is needed only in non-angular applications. For angular appplications it can be directly provided with the help of providers. If the value of bpmn diagram is required it can passed to web component in the manner as described below. This can be used to dynamically use the XML when working with the sourceloop-workflow web component. 
```

Comment:
specify that diagram attribute needs to be set 
justifying your added piece of code in example below